docs/meetings/team/2023_03_27.md

```diff
@@ -0,0 +1,70 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 10:05am
+
+**End Time** 10:25am
+
+**Location** Discord
+
+**Date** March 27, 2023
+
+## 👋 Attendence
+
+- Veronica
+- Edouard
+
+Attendance Notes:
+
+Matt and Logan were unable to attend today
+
+## 🃏 Joke of the Day
+
+I have a brand new whiteboard, it's quite remarkable.
+
+## 📢 Announcements
+
+- This week:
+  - GitStats Week 25
+- Upcoming:
+  - Promo video (to show during final presentation) due April 7
+  - Final presentation slides due April 7
+  - Our Final Presentation is at 3:45 pm on April 12
+  - Wings Wednesday on April 12
+  - Peer Evaluation IV due Apr 14
+  - Final Report due Apr 14
+
+## 📅 Last Week (Restrospective)
+
+- Peer testing report: we found some issues we can work on this week
+- We're satisfied with our video generation time
+- We were informed we won't receive anymore assets from the Kukarella artist
+
+## 📋 This Week
+
+Peer Testing II issues found
+
+- Blinking probability changes (Veronica)
+- Documentation for server-side AWS implementation (Matt & Ed)
+- Documentation links to tooling & standard error codes
+- Documentation for tooling (V added screenshots in the issue for Logan, but isn't sure which screenshots he wanted)
+- Documentation for how to add transition frames (Veronica)
+- Documentation to say where new character image assets should go
+
+Brainstorming for the promo video
+
+- we could have Barb or Boy speak the lines
```

Comment:
I think this is a good idea, might require some extra video editing but I think we could overlay them on a screen recording and remove the white background too